lib/events/stream.go

```diff
@@ -494,7 +485,7 @@ func (w *sliceWriter) receiveAndUpload() {
 					return
 				}
 			}
-			defer w.completeStream()
```

Comment:
I believe this is bogus at exit unless it explicitly needs to add to the defer list (which I don't think is necessary).